plugins/juju/README.md

```diff
@@ -0,0 +1,107 @@
+| jrs   | juju remove-storage                                    | Remove storage                                                            |
+| jrs!  | juju remove-storage --force                            | Remove storage even if it is currently attached                           |
+
+
+### Relation
+| Alias  | Command                                               | Description                                                               |
+|--------|-------------------------------------------------------|---------------------------------------------------------------------------|
+| jrel   | juju relate                                           | Relate two applications                                                   |
+| jrmrel | juju remove-relation                                  | Remove an existing relation between two applications.                     |
+
+### Cross-model relation (CMR)
+| Alias  | Command                                               | Description                                                               |
+|--------|-------------------------------------------------------|---------------------------------------------------------------------------|
+| jex    | juju expose                                           | Make an application publicly available over the network                   |
+| jof    | juju offer                                            | Offer application endpoints for use in other models                       |
+| jcon   | juju consume                                          | Add a remote offer to the model                                           |
+| jrmsas | juju remove-saas                                      | Remove consumed applications (SAAS) from the model                        |
+| junex  | juju unexpose                                         | Remove public availability over the network for an application            |
+
+### Bundle
+| Alias | Command                                                | Description                                                               |
+|-------|--------------------------------------------------------|---------------------------------------------------------------------------|
+| jeb   | juju export-bundle                                     | Export the current model configuration as a reusable bundle               |
+
+## Functions
+
+| Function | Arguments                         | Description                                                |
+|----------|-----------------------------------|------------------------------------------------------------|
+| `jaddr`  | `app_name [unit_num]              | displau app or unit ip address                             |
+| `jreld`  | `relation_name app_name unit_num` | display app and unit relation data                         |
+| `wjst`   | `[interval]`                      | watch juju status, with optional interval (default: 5 sec) |
```

Comment:
The function accepts more arguments that are passed to the `watch` commands, you should document that or fix the function code.